src/components/MissionArea/MissionArea.js

```diff
@@ -3,20 +3,33 @@ import './MissionArea.css';
 const MissionArea = (props) => {
   let {currentCountry} = props
   console.log(currentCountry)
+  let takeOver;
+  let languages;
+  if(!currentCountry.regionalBlocs[0]){
+    takeOver = currentCountry.region
+  }else{
+    takeOver = `the ${currentCountry.regionalBlocs[0].name}`
+  }
+  if(!currentCountry.languages[1]){
+    languages = currentCountry.languages[0].name
+  } else{
+    languages = `${currentCountry.languages[0].name} and ${currentCountry.languages[1].name}`
```

Comment:
This conditional rendering looks good. It takes care of the small details in our mission area